tests/functional/python/test_resolvers_python.py

```diff
@@ -129,7 +129,6 @@ def get_dependencies(self, candidate):
 
 XFAIL_CASES = {
     "pyrex-1.9.8.json": "Too many rounds (>500)",
-    "same-package-extras.json": "State not cleaned up correctly",
```

Comment:
I am wondering about this. The change above caused this test to fail but I'm afraid I don't understand the reason it was marked XFAIL in the first place. Is my fix even valid?

Reply:
It could be that the issue was fixed; we did make some improvements around the area a while ago. The test case _should_ pass, so it’s a good idea to remove this xfail if it does.